arlas-rest/src/main/java/io/arlas/server/rest/explore/search/SearchRESTService.java

```diff
@@ -299,6 +308,7 @@ public Response searchPost(
         MixedRequest request = new MixedRequest();
         request.basicRequest = search;
         request.headerRequest = searchHeader;
+        RequestColumnFilter.checkRequest(columnFilter, collectionReference, search, searchHeader);
```

Comment:
I think we should add the fields in `column-filter` into the `include` parameter

Reply:
Indeed, that's done